### ISSUE TITLE
Find ideal learning rate values

### ISSUE THREAD

**Issue by dbeinhauer** [dbeinhauer/mcs-source#35]
After changes made on the model to perform optimizer step after each time step the training now converges much faster but does not reach as good score as it was reaching before (0.76 now in comparison to 0.86 before). The problem might be connected to not ideally set learning rate. Inspect the various range of possible learning rates to figure out the best one.

**Comment by dbeinhauer** [comment]
It seems that ideal learning rate is in interval (0.00001; 0.000007). The results might be better using slightly smaller learning rates (0.000007; 0.000005) in case of hidden RNN steps.
